projects/bp-gallery/src/components/views/archives/ArchiveLogoInput.tsx

```diff
@@ -35,10 +34,7 @@ const ArchiveLogoInput = ({ defaultUrl, onChange }: ArchiveLogoInputProps) => {
           <div>
             {t('archives.edit.logo.preview')}
             <div className='archive-logo-container'>
-              <img
-                className='archive-logo'
-                src={logo ? URL.createObjectURL(logo) : asApiPath(`/${defaultUrl}`)}
-              />
+              <img className='archive-logo' src={logo ? URL.createObjectURL(logo) : defaultUrl} />
```

Comment:
But in this case the potential memory leak could not become problematic, since we only generate 1 objecturl per archive, right? Or could multiple objecturls be created for the same archive?

Reply:
A new object URL is created every time an `ArchiveLogoInput` is rerendered. Because of #485 (which I discovered because of your question, so thank you), this currently happens every five seconds. So the potential for a memory leak is actually there 😄